packages/test-kit/src/with-feature.ts

```diff
@@ -443,18 +444,32 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}): WithF
                 if (dependencies) {
                     if (dependencies.type === 'link') {
                         linkNodeModules(projectPath, dependencies.path);
-                    } else if (dependencies.type === 'yarn') {
-                        spawnSyncSafe('yarn', ['install'], {
+                    } else if (dependencies.type === 'yarn' || dependencies.type === 'npm') {
+                        const installOptions = {
                             cwd: projectPath,
                             shell: true,
                             stdio: debugMode ? 'inherit' : 'ignore',
-                        });
-                    } else if (dependencies.type === 'npm') {
-                        spawnSyncSafe('npm', ['install'], {
-                            cwd: projectPath,
-                            shell: true,
-                            stdio: debugMode ? 'inherit' : 'ignore',
-                        });
+                        } satisfies SpawnSyncOptions;
+                        const shouldRetryInstall = !!process.env.NEW_FIXTURE_DEPENDENCIES_INSTALLATION;
+
+                        if (shouldRetryInstall) {
+                            const installTimeout = 10_000;
+
+                            await retry(
```

Comment:
I think the test time might needs to be adjusted when successful
see adjustCurrentTestTimeout in core3utils/testing